### ISSUE TITLE
test fails with numpy 1.11

### ISSUE THREAD

**Issue by ignatenkobrain** [alexandrebarachant/pyRiemann#20]
```python
======================================================================
ERROR: Test inverse transform of Tangent Space without fit.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/builddir/build/BUILD/python-pyriemann-0.2.3/python2/tests/test_tangentspace.py", line 68, in test_TangentSpace_inversetransform_without_fit
    ts.inverse_transform(tsv)
  File "/builddir/build/BUILDROOT/python-pyriemann-0.2.3-4.fc24.noarch/usr/lib/python2.7/site-packages/pyriemann/tangentspace.py", line 178, in inverse_transform
    self._check_reference_points(X)
  File "/builddir/build/BUILDROOT/python-pyriemann-0.2.3-4.fc24.noarch/usr/lib/python2.7/site-packages/pyriemann/tangentspace.py", line 110, in _check_reference_points
    self.Cr = numpy.eye(self._check_data_dim(X))
  File "/usr/lib64/python2.7/site-packages/numpy/lib/twodim_base.py", line 233, in eye
    m = zeros((N, M), dtype=dtype)
TypeError: 'numpy.float64' object cannot be interpreted as an index
----------------------------------------------------------------------
```

**Comment by alexandrebarachant** [comment]
thanks for the bug report.
Did any other tests fail ?

**Comment by ignatenkobrain** [comment]
No one with released 0.2.3
